client/catalog/src/app/App.svelte

```diff
@@ -98,8 +100,16 @@
 </style>
 <svelte:head>
 	<title>Aft</title>
-	<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
-
+	<link rel="stylesheet"
+		href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700&display=swap">
+	<link rel="stylesheet"
+		href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.52.2/codemirror.min.css">
+	<link rel="stylesheet"
```

Comment:
I don't really love linking to these stylesheets/the component JS. 

Ideally we'd be able to bundle them into our application. I'm OK merging this for now because I want you to keep your momentum on the project, but let's figure out how to do that before too long.